gradle/versioning.gradle

```diff
@@ -28,14 +28,14 @@ tasks.register("validateVersionMatchesTag") {
         }
 
         if (gitTag != project.version.toString()) {
-            throw new Exception("The tag for this commit is '$gitTag', but the project version is set to '$project.version'.")
+            throw new Exception("The tag for this commit is \"$gitTag\", but the project version is set to \"$project.version\".")
```

Comment:
Same as above - probably fine to keep these as single quotes

Reply:
Addressed in next series of commits pushed to this PR.